templates/vue-demo-store/pages/account/order.vue

```diff
@@ -33,6 +97,10 @@ await useAsyncData("getOrders", () => {
     ],
   });
 });
+
+await useAsyncData("getTotalOrders", async () => {
+  loadTotalOrdersCount();
+});
```

Comment:
we're already loading orders, so total count should already be available

Reply:
That is false because the total count differs when you set a limit.
But this could also be a bug in the admin API endpoint.

Reply:
sounds like a bug in the API or criteria differs, the total should only change when the criteria changes, the limit cannot change the `total`